src/Mvc/Mvc.Abstractions/src/PublicAPI.Unshipped.txt

```diff
@@ -51,7 +51,7 @@ Microsoft.AspNetCore.Mvc.ActionConstraints.ActionConstraintContext.Candidates.ge
 Microsoft.AspNetCore.Mvc.ActionConstraints.ActionConstraintContext.Candidates.set -> void
 Microsoft.AspNetCore.Mvc.ActionConstraints.ActionConstraintContext.CurrentCandidate.get -> Microsoft.AspNetCore.Mvc.ActionConstraints.ActionSelectorCandidate
 Microsoft.AspNetCore.Mvc.ActionConstraints.ActionConstraintContext.CurrentCandidate.set -> void
-Microsoft.AspNetCore.Mvc.ActionConstraints.ActionConstraintContext.RouteContext.get -> Microsoft.AspNetCore.Routing.RouteContext!
+Microsoft.AspNetCore.Mvc.ActionConstraints.ActionConstraintContext.RouteContext.get -> RouteContext!
```

Comment:
On second thought, I'm not sure why these lines are changing. Suggest running through the update steps I described in email using `src/Mvc/startvs.cmd` instead. Something may be missing.